docs/compiling.rst

```diff
@@ -206,6 +208,57 @@ information about usage in C++, see :doc:`/advanced/embedding`.
     add_executable(example main.cpp)
     target_link_libraries(example PRIVATE pybind11::embed)
 
+.. _building_manually:
+
+Building manually
+=================
+
+pybind11 is a header-only library, hence it is not necessary to link against
+any special libraries and there are no intermediate (magic) translation steps.
+
+On Linux, you can compile an example such as the one given in
+:ref:`simple_example` using the following command:
+
+.. code-block:: bash
+
+    $ c++ -O3 -Wall -shared -std=c++11 -fPIC `python3 -m pybind11 --includes` example.cpp -o example`python3-config --extension-suffix`
```

Comment:
This assumes that pybind11 has been installed using pip or conda, which may be worth mentioning.

Reply:
If it hasn't, this needs to be the Python include directory (`python-config3 --includes`) and PyBind's include directory, as that command incorporates both.

Reply:
This should be addressed by the latest commit.